Formula/httpstat.rb

```diff
@@ -0,0 +1,16 @@
+class Httpstat < Formula
+  desc "Curl statistics made simple"
+  homepage "https://github.com/reorx/httpstat"
+  url "https://github.com/reorx/httpstat/archive/v1.1.2.tar.gz"
+  sha256 "0468abebfbae9ff7ea35d577b60ff9af9db3ff67ffd81b11ac005c4e9602919b"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    bin.install "httpstat.py" => "httpstat"
+  end
+
+  test do
+    assert_match "HTTP/1.1", shell_output("#{bin}/httpstat https://github.com")
```

Comment:
Well, then I think match `HTTP/` is better, but for `200 OK`,  I really think we shouldn't involved github's status into our contest of testing a program that has no relationship with github service, even though github is the fundamental dependency of brew service in practice, but it is not the requirement of httpstat program. So following the principle in testing concept that unrelated factor shouldn't be involved, we just need to see:

1. Whether httpstat exit with 0
2. Whether httpstat output a response of HTTP protocol

is enough to determine if httpstat is successfully installed.